app/weather_api.rb

```diff
@@ -0,0 +1,27 @@
+require 'sinatra'
+require 'faraday'
+require 'json'
+require 'pry'
+# require_relative './config/application'
+
+
+# class WeatherApi < Sinatra::Base
+
+
```

Comment:
Should this endpoint be hard coded as 'new york'?  Could it just be :location in the endpoint and we pass the location value

Reply:
We are actually passing the lat and lon now. This is an old version.